extra-rocm/rocm-bandwidth-test/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=rocm-bandwidth-test
+PKGDES="Utility to measure the GPU application memory bandwidth related performance"
+PKGSEC=devel
+PKGDEP="rocr-runtime"
+BUILDDEP="cmake"
+
+CMAKE_AFTER="-DCMAKE_INSTALL_PREFIX=/opt/rocm -DCMAKE_PREFIX_PATH=/opt/rocm -DCMAKE_SKIP_RPATH=OFF -DCMAKE_INSTALL_RPATH=\$ORIGIN/../lib"
```

Comment:
Where did `$ORIGIN` come from and, does this have to be installed in `/opt`?

Reply:
It's some special grammar of rpath.

Reply:
I see. And @Icenowy noted on Telegram that we could be facing more challenge than it's worth if we force it to be installed in `/usr`, even if it is a requirement in our Styling Manual.